src/MyEventEmitter.js

```diff
@@ -1,12 +1,83 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
-}
+const myEventListener = class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, listener) {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+
+    this.events[eventName].push(listener);
+
+    return this;
+  }
+
+  once(eventName, listener) {
+    if (this.events[eventName]) {
+      this.events[eventName] = listener;
```

Comment:
Method `once` means that I want to call a specific `listener` only once for a specific event, but it doesn't mean that this specific listener can overwrite other listeners for the same event that were created, for example, by `on` method. I think you need to wrap a listener with some function that calls the listener and then clears it from the events queue.